siliconcompiler/report/streamlit_viewer.py

```diff
@@ -708,14 +851,14 @@ def make_metric_to_metric_unit_map(metric_dataframe):
     # setting up tabs
     if 'flowgraph' not in streamlit.session_state:
         streamlit.session_state['flowgraph'] = True
+    tabs = ["Metrics", "Manifest", "File Viewer", "Graphs"]
```

Comment:
lets only show the graphs tab is there is something to graph (ie. more than one history item)